src/eq_schema/Answer.test.js

```diff
@@ -16,126 +17,145 @@ describe("Answer", () => {
       answer
     );
 
-  it("should generate  a valid eQ answer from an author answer", () => {
-    const answer = new Answer(createAnswerJSON({ type: "PositiveInteger" }));
+  // it("should generate  a valid eQ answer from an author answer", () => {
```

Comment:
This looks like you have just commented out a test?

Reply:
@jonnyshaw89 Yeah good spot. Forgot to uncomment the tests back in. Should now be uncommented.